site/docs/resources/faq.md

```diff
@@ -39,6 +39,20 @@ If you're coming from a different programming language or framework, you can che
 
 ## Why Am I Getting This Error?
 
+### 400 Bad Request: Cannot parse entities
```

Comment:
If we capitalized first letter in here, the following heading should do the same to.

Reply:
I applied exactly the same casing as in the original error messages, even though I agree that this is debatable. We could apply proper title case as everywhere else in the docs. What would you prefer?

Reply:
I thought this was a comment you wrote, if this is the original message then we should keep it.

Reply:
Alright!

Reply:
All headers are the original error messages, so I think it may also make sense to keep them that way across translations.